coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 98.66
```

Comment:
Testing coverage is still above 95%; my goal is achieve 100% coverage once all iterations are complete, and in the mean time to keep it above 95%